### PR TITLE
137 UI tests

### PR DESCRIPTION
**Pull request by domi1504**
Added more UI-Tests for every page.
Currently only testing basic appearance, we did not find any possibility of testing workflows like adding a device, since mocking of e.g. the bluetooth service is not really possible.

### CODE REVIEW

**Comment by domi1504** [comment]
ATTENTION: PLEASE DELETE THIS PR. ACCIDENTALLY PR INTO MAIN, NOT DEV.

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj [hunk 1]
```diff
@@ -5,12 +5,17 @@
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugType>full</DebugType>
+    <DefineConstants>TRACE;DEBUG;NETSTANDARD;NETSTANDARD2_0;ENABLE_TEST_CLOUD;</DefineConstants>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+  </PropertyGroup>
```
Comment (Chippelius) [inline Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj:12]: Is this change necessary / sensible?

Comment (domi1504) [reply Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj:12]: Yes. It is a necessary step for UI testing on iOS, as per official docs
